newsroom/flaskapp.py

```diff
@@ -193,6 +199,11 @@ def _setup_theme(self):
             view_func=self.send_theme_file
         )
 
+    def _setup_celery_serialization(self):
```

Comment:
Do we need this if we register the serializer with different name i.e. `newsroom/json`?

Reply:
Nope, I will remove it.